packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -370,21 +380,23 @@ Media.toggleStream = () => {};
 
 /**
  * Stop input stream
- * @param {MediaTrack} track A media stream
+ * @param {LocalStream} stream A local stream
  * @returns {null}
  */
-Media.stopTracks = (track: any) => {
-  if (!track) {
+Media.stopStreams = (stream: LocalStream) => {
+  if (!stream) {
     return Promise.resolve();
   }
 
   return Promise.resolve().then(() => {
-    if (track && track.stop) {
+    if (stream && stream.stop) {
       try {
-        track.stop();
+        stream.stop();
       } catch (e) {
         LoggerProxy.logger.error(
-          `Media:index#stopTracks --> Unable to stop the track with state ${track.readyState}, error: ${e}`
+          `Media:index#stopTracks --> Unable to stop the track with state ${
```

Comment:
not sure if this log makes sense..... it should say "stream" not "track" and why is it logging the readyState of only the output stream's track? it should probably log it for both inputTrack and outputTrack

Reply:
Ok. Will add a log for inputStream as well and update the track to stream. Probably a missed one while renaming.

Reply:
Done in #3053